src/main/webapp/authentication-app/src/firebase.js

```diff
@@ -0,0 +1,40 @@
+import firebase from "firebase/app";
+import "firebase/auth";
+import "firebase/firestore";
+import "firebase/analytics";
+
+const firebaseConfig = {
+    apiKey: "AIzaSyAjse2YjCJHUUe5w1i7I9jkjH8cqqRcAIU",
```

Comment:
You don't want to put an apiKey into github. Instead I would create a separate file that you can import it from. Also include that new file in your .gitignore file